src/viam/services/slam/slam.py

```diff
@@ -83,7 +83,8 @@ async def get_properties(self, *, timeout: Optional[float]) -> Tuple[bool, Mappi
             slam_properties = await slam_svc.get_properties()
 
         Returns:
-            Tuple[bool, MappingMode.ValueType]: A tuple of a boolean value representing if the SLAM session is being run in
-            the cloud and the mapping mode of said session
+            Tuple[bool, MappingMode.ValueType, str, List[SensorInfo]]: A tuple containing: A boolean value representing if the SLAM session
+            is being run in the cloud, the mapping mode of said session, a string with the internal state file type of the SLAM algorithm,
+            and a list containing information about the sensors used for the SLAM session.
         """
```

Comment:
There doesn't seem to be a standard with how we introduce tuples in doc strings, but maybe it would be better to use this format used in [arm's get_kinematics](https://github.com/viamrobotics/viam-python-sdk/blob/e4dd87b490782f07ce6c3dc7b159d11aaed60333/src/viam/components/arm/arm.py#L194). Not a necessity, but wanted to provide an alternative in case there was a preference.

Reply:
Updated it! Let me know what you think.